libs/sdk-react-native/ios/RNBreezSDK.swift

```diff
@@ -356,10 +356,10 @@ class RNBreezSDK: RCTEventEmitter {
         }
     }
     
-    @objc(receiveOnchain:rejecter:)
-    func receiveOnchain(_ resolve: @escaping RCTPromiseResolveBlock, rejecter reject: @escaping RCTPromiseRejectBlock) -> Void {
+    @objc(receiveOnchain:resolver:rejecter:)
+    func receiveOnchain(_ req:[String: Any], resolver resolve: @escaping RCTPromiseResolveBlock, rejecter reject: @escaping RCTPromiseRejectBlock) -> Void {
         do {
-            let swapInfo = try getBreezServices().receiveOnchain()
+            let swapInfo = try getBreezServices().receiveOnchain(req: req)
             resolve(BreezSDKMapper.dictionaryOf(swapInfo: swapInfo))        
         } catch let err {
             rejectErr(err: err, reject: reject)
```

Comment:
The req needs to be converted into a ReceiveOnchainRequest type
```
        if let receiveOnchainRequest = BreezSDKMapper.asReceiveOnchainRequest(reqData: req) {
            do {
                let swapInfo = try getBreezServices().receiveOnchain(req: receiveOnchainRequest)                
                resolve(BreezSDKMapper.dictionaryOf(swapInfo: swapInfo))
            } catch let err {
                reject(RNBreezSDK.TAG, "Error calling receiveOnchain \(err)", err)
            }
        } else {
            rejectErr(err: SdkError.Generic(message:"Invalid reqData"), reject: reject)            
        }
```

Reply:
Done as part of 4efe618